dronegen/tag.go

```diff
@@ -308,6 +319,34 @@ func tagCopyPackageArtifactCommands(b buildType, packageType string) []string {
 	return commands
 }
 
+// createReleaseAssetCommands generates a set of commands to create release & asset in release management service
+func tagCreateReleaseAssetCommands(b buildType) []string {
+	commands := []string{
+		`WORKSPACE_DIR=$${WORKSPACE_DIR:-/}`,
+		`VERSION=$(cat $WORKSPACE_DIR/go/.version.txt)`,
+		fmt.Sprintf(`RELEASES_HOST=%v`, releasesHost),
+		`echo $RELEASES_CERT | base64 -d > $WORKSPACE_DIR/releases.crt`,
+		`echo $RELEASES_KEY | base64 -d > $WORKSPACE_DIR/releases.key`,
+		`trap "rm -f /releases.crt /releases.key" EXIT`,
+		`CREDENTIALS="--cert $WORKSPACE_DIR/releases.crt --key $WORKSPACE_DIR/releases.key"`,
+		`which curl || apk add --no-cache curl`,
```

Comment:
Don't `apk add` need some sort of `-y` flag to not block on input?

Reply:
I don't think so, apk does not seem to ask for input in normal circumstances:

```console
$ docker run -it alpine:latest
/ # apk add --no-cache curl
fetch https://dl-cdn.alpinelinux.org/alpine/v3.13/main/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.13/community/x86_64/APKINDEX.tar.gz
(1/5) Installing ca-certificates (20191127-r5)
(2/5) Installing brotli-libs (1.0.9-r3)
(3/5) Installing nghttp2-libs (1.42.0-r1)
(4/5) Installing libcurl (7.79.1-r0)
(5/5) Installing curl (7.79.1-r0)
Executing busybox-1.32.1-r6.trigger
Executing ca-certificates-20191127-r5.trigger
OK: 8 MiB in 19 packages
```

Reply:
It looks like, if you want interactive, you have to pass `apk add -i`, so it's kind of the opposite of others. Thanks for looking it up.